src/extension/styling-definitions/styling-definition.js

```diff
@@ -0,0 +1,220 @@
+import { fontResolver as createFontResolver } from "qlik-chart-modules";
+import DEFAULTS from "../../style-defaults";
+import propertyResolver from "../../utils/property-resolver";
+import labelStylingDefinition from "./styling-utils";
+
+const colorOptions = [
+  { value: "auto", translation: "Common.Auto" },
+  { value: "colorPicker", translation: "properties.colorMode.primary" },
+  { value: "byExpression", translation: "properties.colorMode.byExpression" },
+];
+
+function createStylingDefinition(theme, flags, translator) {
+  const fontResolver = createFontResolver({
+    theme,
+    translator,
+    flags,
+    config: {
+      id: "orgChart",
+      paths: ["axis.label.name", "label.value"],
+    },
+  });
+
+  const bordersActive = (data) =>
+    (propertyResolver.getValue(data, "border.top") ?? DEFAULTS.BORDER_TOP) ||
+    (propertyResolver.getValue(data, "border.fullBorder") ?? DEFAULTS.BORDER_FULL);
```

Comment:
If `propertyResolver.getValue(data, "border.top")` returns something falsey, doesn't the function always return `DEFAULTS.BORDER_TOP`? So the second part after `||` isn't needed?

Reply:
The ?? is returning the first not null/undefined value, so it will still return false if the value for border.top is false https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
ahh, I didn't know that. I thought it was for false as well, good to know

Reply:
Do you need the propertyResolver here? Would `data?.border?.top` not be enough?

Reply:
Also to clarify. The intention is that if the top border is `0`, it should fallback to using the full border?

Reply:
The bordersActive function is only used to show/hide the color dropdown selector for the border. The selector will be displayed if at least one of them is true (top or fullBorder). 
If top is 0, it doesn't affect the full border value. They are independent of each other.

Reply:
I see. I was not expecting them to be booleans.